### ISSUE TITLE
Task Stats appear to show incorrect figure for 'needed more mapping' under tasks

### ISSUE THREAD

**Issue by rtbk91** [hotosm/tasking-manager#3666]
It looks as though the 'needed more mapping' count is adding on some tasks which the user when validating has marked as invalid or something else. An example is my user account should say 5 but has 459 (rtbk).

**Comment by willemarcel** [comment]
The goal of that information is to show how the tasks mapped by the user were evaluated by validators. So it doesn't contain the number of tasks you mapped that, at the current moment, needs more mapping, but the number of tasks you mapped and a validator found it needed more mapping.

**Comment by rtbk91** [comment]
Yes I agree but I haven't had needed more mapping back on over 400, just
the 5 in my inbox.

**Comment by rtbk91** [comment]
![Capture](https://user-images.githubusercontent.com/72060263/96653802-b0a34400-1385-11eb-8098-a24fa72f720e.PNG)
I would think that the sum in the tasks section should be: 
Total Tasks= Validated + needed more mapping + Un-validated
The sum here is greater than the total mapped.

There are also some users that have greater tasks validated than that they have mapped
